its/ruling/src/test/resources/expected/python-S6911.json

```diff
@@ -1,10 +1,6 @@
 {
 "project:tensorflow/python/keras/integration_test/function_test.py": [
 168,
-191,
-221
-],
-"project:tensorflow/python/keras/integration_test/saved_model_test.py": [
-164
+191
```

Comment:
What about those remaining issues? Again, if it's not TP, we should address or acknowledge it somehow. For example with a unit test, even if it's a documented FP we don't address (or just a bordeline case we actually want to raise an issue on - but the intention should be clear).

Reply:
I have added some extra logic to handle some cases, but I think the implementation is open to discussion